compiler/rustc_ast_pretty/src/pprust/state.rs

```diff
@@ -377,6 +378,10 @@ pub trait PrintState<'a>: std::ops::Deref<Target = pp::Printer> + std::ops::Dere
         self.word(lit.token_lit.to_string())
     }
 
+    fn print_included_bytes(&mut self, n: usize, path: &PathBuf) {
+        self.word(format!("/* {n} bytes included from `{}`*/", path.display()))
```

Comment:
Pretty printing must be precise and use the canonical token representation https://github.com/rust-lang/rust/pull/103812#issuecomment-1310481177.

Reply:
The reason is that pretty-printing results are occasionally used by proc macros, and some tools also compile pretty-printed code in practice.

Reply:
Would it be better to print it as an array (`[1, 2, 3]`) or an escaped byte string (`b"\x01\x02\x03"`), as it currently is? The former is more time/memory intensive for large blobs but looks more readable to me

Reply:
>EDIT: May have misunderstood, should the output instead be the original invocation (`include_bytes!(...)`)?

No, no, I was talking about the expanded form.

>Would it be better to print it as an array (`[1, 2, 3]`) or an escaped byte string (`b"\x01\x02\x03"`), as it currently is? 

Let's keep whatever was printed previously for now.